axelor-human-resource/src/main/java/com/axelor/apps/hr/rest/ExpenseLineRestController.java

```diff
@@ -144,7 +152,10 @@ public Response updateExpenseLine(
     RequestValidator.validateBody(requestBody);
     ExpenseLine expenseLine =
         ObjectFinder.find(ExpenseLine.class, expenseLineId, requestBody.getVersion());
-
+    ExpenseLineToolService expenseLineToolService = Beans.get(ExpenseLineToolService.class);
+    List<Employee> employeeList =
+        expenseLineToolService.filterInvitedCollaborators(
+            requestBody.fetchInvitedCollaboratorList(), expenseLine, expenseLine.getExpenseDate());
```

Comment:
this code should be moved to `ExpenseLineUpdateService`, the code in the controller must only parse the query and call the right service.